addons/core/package.json

```diff
@@ -50,45 +51,45 @@
   },
   "devDependencies": {
     "@babel/core": "^7.26.10",
```

Comment:
looks like the ember upgrade added `@babel/core` to the `dependencies` section as well

Reply:
this part is important 👍🏻 https://github.com/ember-cli/ember-addon-output/blob/v5.12.0/package.json#L31

Reply:
Interesting! it add it to dependencies (v7.25.2), but skipped to me that was also present in devDependencies with a newer version v7.26.10.
Since your PR for addons/auth is using the v7.26.10, for consistency I will keep only the v7.26.10 within dependencies.
Thank you for the catch!!